src/components/ButtonToReset/ButtonToReset.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'semantic-ui-react';
+
+export const ButtonToReset = React.memo(
+  ({ reset }) => (
+    <Button basic color="pink" content="Reset" onClick={reset} />
```

Comment:
Форматуємо так само як і в атрибути в HTML-i
```suggestion
    <Button 
      basic 
      color="pink" 
      content="Reset" 
      onClick={reset} 
    />
```